src/drive/web/modules/drive/Toolbar/components/CreateNoteItem.jsx

```diff
@@ -25,7 +40,7 @@ const CreateNoteItem = ({ client, t, displayedFolder }) => {
           window.location.href = await models.note.generatePrivateUrl(
```

Comment:
Is it ok to use `window.location.href` for the redirection? Shouldn't we use `open(url)` or something similar?

Reply:
Yep, it's ok. Our apps are not configured to handle this kind of opening, so it opens the browser.